backend/TraderX/src/main/java/cmpe451/group6/rest/annotation/controller/AnnotationController.java

```diff
@@ -0,0 +1,93 @@
+package cmpe451.group6.rest.annotation.controller;
+
+import cmpe451.group6.Util;
+import cmpe451.group6.rest.annotation.model.Annotation;
+import cmpe451.group6.rest.annotation.service.AnnotationService;
+import cmpe451.group6.rest.annotation.dto.AnnotationDTO;
+import cmpe451.group6.authorization.dto.StringResponseWrapper;
+
+import io.swagger.annotations.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+import java.util.Map;
+
+import static cmpe451.group6.authorization.exception.GlobalExceptionHandlerController.GENERIC_ERROR_RESPONSE;
+
+@RestController
+@RequestMapping("/annotation")
+@Api(tags = "Annotation")
+public class AnnotationController {
+
+    @Autowired
+    AnnotationService annotationService;
+
+    @Autowired
+    private Util util;
+
+    @GetMapping("{id}")
```

Comment:
Is a slash needed here? I mean, is the endpoint `annotation/1` or  `annotation1` for this form?

Reply:
`annotation/1` in this form. Most probably system auto handles this issue. I checked it out as `annotation/1` and no problems

Reply:
`annotation/1` in this form. Most probably system auto handles this issue. I checked it out as `annotation/1` and no problems